pkg/k8shandler/clusterloggingrequest.go

```diff
@@ -16,7 +16,7 @@ import (
 
 type ClusterLoggingRequest struct {
 	client  client.Client
```

Comment:
while we're at it, why not also rename this and change its case? something like `K8sClient`? (since `Client` is very overloaded)